examples/redis_repl/store/db_test.go

```diff
@@ -26,34 +29,49 @@ func TestMain(m *testing.M) {
 }
 
 func initRedisStore() {
-	if rs, err := NewRedisDB(redisHost, redisPort, stats.NewNoOpClient()); err != nil {
+	if rs, err := NewRedisDB(redisHost, redisPort, metadataDB, stats.NewNoOpClient()); err != nil {
 		panic(err)
 	} else {
 		store = rs
 	}
 }
 
 func insertKey(t *testing.T, key, val string, dbIdx int) {
-	saveReq := fmt.Sprintf("redis.call('select', '%d') return redis.call('set', '%s', '%s')", dbIdx, key, val)
-	if saveRes, err := store.Save([]byte(saveReq)); err != nil {
+	saveReq := fmt.Sprintf("return redis.call('set', '%s', '%s')", key, val)
+	req := &api.SaveRequest{
+		Data: []byte(saveReq),
+		Args: map[string][]byte {DBIndexKey: []byte(strconv.Itoa(dbIdx))},
```

Comment:
Wouldn't having a proper `args` object instead of `map[string]byte` be better?

Reply:
We wanted to leave this as an arbitrary collection of tuples that the DB implementations choose. Hence at the Nexus level it has to remain as such.